FreeRTOS-Plus/Source/Application-Protocols/network_transport/using_mbedtls/using_mbedtls/using_mbedtls.c

```diff
@@ -527,8 +527,8 @@ static TlsTransportStatus_t tlsHandshake( NetworkContext_t * pNetworkContext,
         /* coverity[misra_c_2012_rule_11_2_violation] */
         mbedtls_ssl_set_bio( &( pTlsTransportParams->sslContext.context ),
                              ( void * ) pTlsTransportParams->tcpSocket,
-                             mbedtls_platform_send,
-                             mbedtls_platform_recv,
+                             MBEDTLS_SSL_SEND,
+                             MBEDTLS_SSL_RECV,
```

Comment:
Good to also leave a comment here that this macro can be defined from `mbedtls_config.h`

Reply:
Please check [this commit](https://github.com/FreeRTOS/FreeRTOS/pull/695/commits/5eea1b83d94bc303205bfb07b439ddf22bfed204).